src/main/java/dumshenko/daniil/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package dumshenko.daniil;
+
+public class Rectangle extends Shape {
+
+    private double rectangleWidth;
+    private double rectangleHeight;
```

Comment:
nit: Here it is clear from the class name that the `width` and `height` refer to the `Rectangle`.
```suggestion
    private double width;
    private double height;
```